src/clients/SparkCore.js

```diff
@@ -68,7 +68,6 @@ const MAX_BINARY_SIZE = 108000; // According to the forums this is the max size.
  */
 class SparkCore extends EventEmitter {
   _cipherStream: ?Duplex = null;
-  _connectionKey: ?string = null;
```

Comment:
Yeah, it is convenient for the logs though... Maybe just have a global counter and use that instead.